commands/usernote.js

```diff
@@ -201,6 +217,23 @@ function formatNote (note, subreddit) {
   const content = note.n;
   return `"${content}" (${timestamp}, by ${author}${link ? `, on link ${link} ` : ''}, colored ${color})`;
 }
+function pushToCache (channel, note) {
+  if (channel in removedNoteCache) {
```

Comment:
The `in` operator in JS is sort of weird. For example, if channel is the string `hasOwnProperty`, then `channel in removedNoteCache` will be `true`, because there is a `hasOwnProperty` function on Object.prototype.

If you create removedNoteCache with `Object.create(null)`, then it won't have functions like `hasOwnProperty`, so we won't have to worry about this.

```js
Object.prototype.hasOwnProperty.call(removedNoteCache)
```